packages/devtools_app/lib/src/memory/memory_filter.dart

```diff
@@ -299,67 +333,75 @@ class SnapshotFilterState extends State<SnapshotFilterDialog>
     //      - one row of buttons Ok/Cancel
     // For very tall app keep the dialog at a reasonable height w/o too much vertical whitespace.
     // The listbox area is the area that grows to accommodate the list of known libraries.
-    // TODO(devoncarew): Convert to a DevToolsDialog.
-    return Dialog(
-      key: snapshotFilterDialogKey,
-      child: LayoutBuilder(
-        builder: (BuildContext context, BoxConstraints constraints) {
-          return Container(
-            width: MediaQuery.of(context).size.width / 3,
-            height: constraints.maxHeight < 400
-                ? constraints.maxHeight
-                : constraints.maxHeight * .3 + (400 * .7),
-            child: Padding(
-              padding: const EdgeInsets.only(left: 15),
-              child: Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
+
+    final constraints = BoxConstraints(
+      maxWidth: defaultDialogWidth,
+      maxHeight: MediaQuery.of(context).size.height,
+    );
+
+    Widget subTitle(String title) => TextField(
```

Comment:
we shouldn't be using a TextField for something that doesn't receive text input. Use `dialogSubHeader`. This will match styling of other dialogs in DevTools

Reply:
If you want it to be selectable for copy and paste you can use SelectableText

Reply:
Using dialogSubHeader, however, I think more things in devtools should be selectable for copy e.g., tables, etc. I'll add an issue.